benten-hackernews-bolt/README.md

```diff
@@ -0,0 +1,271 @@
+# Hackernews Bolt
+
+## Overview
+This bolt provides basic integration with the Hackernews API. The api itself is not very robust and
+is a little unusual when it comes to standard apis. The data that Hackernews exposes is basically just 
+a dump of their in-memory data structures into a firebase firestore database, more info 
+[here](https://github.com/HackerNews/API).
+
+Hackernews's API is only capable of returning a single item via its `id` or a list of `id`s that you can then use to
+query for said `id`s content. Because of the nature of this API I implemented a multithreaded solution that will
+parallelize multiple queries to fetch items from the API via their `id`s. Ideally this service could be abstrscted and
+made useful to any and all other bolts in this project.
+
+## Contextual Information (Definitions)
+- Collection: A list or array of one of the core types of content that hackernews' api returns.
+I.e a list of `topstories`, `newstories`, `beststories`, `askstories`, `showstories`, `jobstories`.
+For more information including how objects are structured in Hackernews's API please see 
+their [API](https://github.com/HackerNews/API). for more information
+
+## Local Development
+To build and run with the Hackernews Bolt active, go through the DialogFlow and Slack setup make sure you have an active benten slack bot running.
+Then run the following commands.
+ 
+1. `cd ~/dev/benten/hackernews-bolt;`
+2. `mvn clean install`
+3. `cd ~/dev/benten/benten-starter;`
+5. `mvn clean install -Dmaven.test.skip=true`
+6. `mvn spring-boot:run;`
+
+If you get an error about the `benten-hackernews-bolt` not being found, try running this:
+
+`cd ~/dev/benten/benten-hackernews-bolt/; mvn install:install-file -Dfile=/Users/jleveroni/dev/benten/benten-hackernews-bolt/target/benten-hackernews-bolt-0.1.5.jar -DgroupId=benten-hackernews-bolt -DartifactId=benten-hackernews-bolt -Dversion=0.1.5 -Dpackaging=jar -DgeneratePom=true;`
```

Comment:
You can remove this section. Since you added packaging as jar a mvn clean install in the parent benten directory should generate the artifact and install it in the m2 repository. Else you can replace specific path variables with place holders.